circuits-lib/Cargo.toml

```diff
@@ -49,4 +48,5 @@ lazy_static = { workspace = true }
 citrea-sov-rollup-interface = { workspace = true }
 
 [features]
-not_patched = []
+default = []
+use-test-vk = []
```

Comment:
would appreciate a short comment here, what's a test-vk (ie. what is the difference of the circuit that uses this vk)